### PR TITLE
fix(avatarChooser): Handle submit without selecting file #38190

### LINKED ISSUE #38190: In settings > avatarChooser , no error message display at frontend for file not selected while saving avatar
### Environment

SaaS (https://sentry.io/)

### Version

_No response_

### Steps to Reproduce

1. got to "https://sentry.io/settings/<org_slug>/" AVATAR section in settings.
2. select Upload an image and do not select any file.
3. Hit save avatar button
4. Here no error message display for telling user he has not selected any file.

### Expected Result

image

### Actual Result

image

### PR DESCRIPTION
**Pull request by growupanand**
Gracefully handle when a user has not selected any file and tries to save Avatar.
While it is handled in backend but also there is need to display error message at frontend.

Fixes #38190

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Comment by evanpurkhiser** [comment]
How about instead we make it so the button is just disabled if you haven't selected anything?

**Comment by growupanand** [comment]
> How about instead we make it so the button is just disabled if you haven't selected anything?

Now save avatar button will disabled if user has not selected any file.

File: static/app/components/avatarChooser.tsx [hunk 1]
```diff
@@ -116,6 +116,11 @@ class AvatarChooser extends Component<Props, State> {
     const {model, dataUrl} = this.state;
 
     ev.preventDefault();
+    // before saving make sure user has selected an file
+    if (dataUrl === null) {
```
Comment (evanpurkhiser) [inline static/app/components/avatarChooser.tsx:120]: Sorry for the late review, we don't need this anymore now that the button is disabled.

Comment (growupanand) [reply static/app/components/avatarChooser.tsx]: Changes Done!

**Comment by evanpurkhiser** [comment]
This looks good, let me make sure all the test are passing and I think we're good to go!

**Comment by growupanand** [comment]
@evanpurkhiser , can you please merge if everything is ok.

**Comment by evanpurkhiser** [comment]
Sorry again for the delay! Will get this merged today!

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**